cfnet/datasets.py

```diff
@@ -154,9 +143,14 @@ class TabularDataModule:
     batch_size: int = 128
     data_name: str = ""
 
-    def __init__(self, data_configs: Dict):
-        # read data
-        self.data = pd.read_csv(Path(data_configs["data_dir"]))
+    def __init__(self, data_configs:dict or str=None):
```

Comment:
Rewrite to 
```python
def __init__(self, data_configs: dict | str=None):
```

See https://birkhoffg.github.io/cfnet/tutorials/contribution.html#write-documentations-in-jupyter-notebook